plugins/presentation/src/components/internal/Properties.svelte

```diff
@@ -13,23 +13,61 @@
 // limitations under the License.
 -->
 <script lang="ts">
-  import type { ClassModel } from '../..'
+  import { ClassModel, getCoreService } from '../..'
   import AttributeEditor from '../AttributeEditor.svelte'
 
   import Icon from '@anticrm/platform-ui/src/components/Icon.svelte'
-
-  // export let _class: Ref<Class<Obj>>
-  // export let excludeAttributes: string[] = []
+  import { Ref, Mixin, Obj } from '@anticrm/core'
 
   export let model: ClassModel
   export let object: any
 
-  // $: getPresentationService()
-  //   .then(service => service.getClassModel(_class))
-  //   .then(m => { model = m.filterAttributes(excludeAttributes) })
+  function toggleVisible (event: MouseEvent) {
+    const table = (event.target as HTMLElement).nextElementSibling as HTMLElement
+    if (table.style.display === 'none') {
+      table.style.display = 'inherit'
+    } else {
+      table.style.display = 'none'
+    }
+  }
+
+  const mixins = model.getMixins().map((_mixin) => {
+    const mixin = {
+      _class: _mixin._mixin
+    } as any
+
+    // filling current object attributes and values
+    const attributes = model.getOwnAttributes(_mixin._mixin).map((_attr) => _attr.key)
+    attributes.forEach((_attr) => {
+      for (const key in object) {
```

Comment:
You should use model.as(object, mixin) and access mixin attribute values, please do not use any ```replace``` logic. Since this logic could be changed.